tests/compval

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import click
+import json
+from jsondiff import diff
+import pandas as pd
+import sys
+
+@click.command()
+@click.argument('json_path', nargs = 1 , required = True)
+def main(json_path):
+
+    #Get/set cv path
+    cv_path = './CMIP6_frequency.json'
+
+    #Open JSON
+    cv = json.load(open(cv_path))
+    j = json.load(open(json_path))
+
+    #Get list of CVS (Need to determine how this information is passed)
+    freq_cv = []
+    for x in cv["frequency"]:
+    #    print(x)
+        freq_cv.append(x)
+    #print(freq_cv)
```

Comment:
This looks good but is hard-coded for frequency. We want it to work for other vocabulary too, e.g. chunk_freq